docs/contributing.md

```diff
@@ -8,7 +8,7 @@
     * follow the style, naming and structure conventions of the rest of the project.
     * make commits atomic and easy to merge.
     * when updating documentation, please see [our guidance for documentation contributions](contributing_docs.md).
-    * when updating the `go.mod` file, please run `make tidy-all` to ensure all modules are updated. Please be aware that the lint stage could fail if this is not done.
+    * when updating the `go.mod` file, please run `make tidy-all` to ensure all modules are updated. It will run `golangci-lint` with the configuration set in the root directory of the project. Please be aware that the lint stage could fail if this is not done.
```

Comment:
Not a blocker, but I notice the use of "please" and future tense; 

- Using "please" is generally discouraged; see here for some more on that https://github.com/docker/cli/pull/5040#issuecomment-2113053344
- In tech docs it's encouraged to use present time  (In this case "It runs ... "); tech docs tends to be in the here and now; if you do this, then this happens (then this "will happen" is both lengthier as well as gives the impression "eventually ... ")

Reply:
TIL

I can change that in a follow-up for all the docs. Wdyt?

Reply:
Definitely! Not important at all. Just sharing some bits I learned from the experts over the Years.

The funny quote in that case was _"There's no future in tech-writing"_

Me: "that sounds really doomy; sure there must be!?"

And then she explained it was about "future tense" 😂😂🤣